docs/src/pinn/parm_estim.md

```diff
@@ -0,0 +1,67 @@
+# Optimising Parameters of a Lorenz System
+ Consider a Lorenz System ,
+! [lorenzSystem](https://raw.githubusercontent.com/ashutosh-b-b/github-doc-images/master/lorenz_system.png) with Physics-Informed Neural Networks.
+Now we would consider the case when we know σ = 10 and we want to optimise β  and ρ.
+We start by defining the the problem,
+```julia
+@parameters t , β, ρ
+@variables x(..), y(..), z(..)
+Dt = Differential(t)
+eqs = [Dt(x(t)) ~ 10*(y(t) - x(t)),
+       Dt(y(t)) ~ x(t)*(ρ - z(t)) - y(t),
+       Dt(z(t)) ~ x(t)*y(t) - β*z(t)]
+
+
+bcs = [x(0) ~ 1.0, y(0) ~ 0.0, z(0) ~ 0.0]
+domains = [t ∈ IntervalDomain(0.0,1.0)]
+dt = 0.1
+
+```
+And then the neural networks as,
+```julia
+input_ = length(domains)
+n = 16
+chain1 = FastChain(FastDense(input_,n,Flux.σ),FastDense(n,n,Flux.σ),FastDense(n,n,Flux.σ),FastDense(n,1))
+chain2 = FastChain(FastDense(input_,n,Flux.σ),FastDense(n,n,Flux.σ),FastDense(n,n,Flux.σ),FastDense(n,1))
+chain3 = FastChain(FastDense(input_,n,Flux.σ),FastDense(n,n,Flux.σ),FastDense(n,n,Flux.σ),FastDense(n,1))
+```
+We will add an additional loss term based on the data that we have in order to optimise the parameters.
+Here we simply calculate the solution of the lorenz system with [OrdinaryDiffEq.jl](https://diffeq.sciml.ai/v1.10/tutorials/ode_example.html#In-Place-Updates-1) and pick up some points randomly
```

Comment:
not randomly: based on the adaptivity of the ODE solver. "This is used to introduce non-uniformity to the time series".